snuba/settings_base.py

```diff
@@ -63,7 +63,7 @@
 RETENTION_OVERRIDES = {}
 
 # the list of keys that will upgrade from a WHERE condition to a PREWHERE
-PREWHERE_KEYS = ['project_id']
+PREWHERE_KEYS = ['issue', 'tags[sentry:release]', 'message', 'environment']
```

Comment:
Please update this to the setting we currently use:
```
['event_id', 'issue', 'tags[sentry:release]', 'message', 'environment', 'project_id']
```

Reply:
Isn't `'tags[sentry:release]'` specific to sentry.io?

Reply:
(just realized this)

Reply:
No. It's the promoted release tag.